### PR TITLE
Responsive config

### PR DESCRIPTION
**Pull request by haringsrob**
Hello @cevou,

This pull request contains a few ideas and code style improvements.

Can you let me know where I can improve?

Regards,

### CODE REVIEW

**Comment by cevou** [comment]
Hi @haringsrob. Thanks for the PR. I will go over it after work.

**Comment by haringsrob** [comment]
@cevou Thanks for the quick reply, I see that one travis test is failing, I will check this tomorrow

**Comment by cevou** [comment]
Can you please give a short description of what all the changes are doing?

File: src/Cevou/Behat/ScreenshotCompareExtension/ServiceContainer/ScreenshotCompareExtension.php [hunk 1]
```diff
@@ -81,55 +83,66 @@ public function configure(ArrayNodeDefinition $builder)
...
-                    ->prototype('array')
-                    ->children()
-                        ->scalarNode('adapter')->defaultValue('default')->end()
-                        ->arrayNode('crop')
-                            ->children()
-                                ->integerNode('left')->defaultValue(0)->min(0)->end()
-                                ->integerNode('right')->defaultValue(0)->min(0)->end()
-                                ->integerNode('top')->defaultValue(0)->min(0)->end()
-                                ->integerNode('bottom')->defaultValue(0)->min(0)->end();
+            ->arrayNode('screenshot_config')
```
Comment (cevou) [inline src/Cevou/Behat/ScreenshotCompareExtension/ServiceContainer/ScreenshotCompareExtension.php:86]: I like the indentation that was here before better. You can see the blocks much easier.

File: src/Cevou/Behat/ScreenshotCompareExtension/ServiceContainer/ScreenshotCompareExtension.php [hunk 2]
```diff
@@ -149,12 +162,13 @@ private function loadAdaptersAndCreateFilesystem(ContainerBuilder $container, ar
...
      * @param ContainerBuilder $container
      * @param AdapterFactory[] $factories
+     *
      * @return string
      * @throws \LogicException
      */
     private function createAdapter($name, array $config, ContainerBuilder $container, array $factories)
     {
-        $adapter = null;
+        $adapter = NULL;
```
Comment (cevou) [inline src/Cevou/Behat/ScreenshotCompareExtension/ServiceContainer/ScreenshotCompareExtension.php:171]: `null` should be lowercase
http://www.php-fig.org/psr/psr-2/

> 2.5. Keywords and True/False/Null
> PHP keywords MUST be in lower case.
> The PHP constants true, false, and null MUST be in lower case.

File: src/Cevou/Behat/ScreenshotCompareExtension/ServiceContainer/ScreenshotCompareExtension.php [hunk 3]
```diff
@@ -20,9 +20,12 @@ class ScreenshotCompareExtension implements ExtensionInterface
     /**
      * @var AdapterFactory[]
      */
-    private $adapterFactories = array();
+    private $adapterFactories = [];
```
Comment (cevou) [inline src/Cevou/Behat/ScreenshotCompareExtension/ServiceContainer/ScreenshotCompareExtension.php:23]: The new array syntax is only supported in PHP 5.4+. Please adjust the composer accordingly.

**Comment by cevou** [comment]
It would also be great if you could split code style improvements and features in different PRs. It's much easier to review then.

**Pull request closed without merging**